src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,102 @@
+import { NavLink } from 'react-router-dom'
+
+import 'css.gg/icons/css/play-button-r.css'
+import 'css.gg/icons/css/film.css'
+
+import Button from '../Button/Button'
+import styled from 'styled-components/macro'
+
+export default function Navigation() {
+  return (
+    <Nav>
+      <NavButton as={NavLink} exact to="/">
+        <HomeIcon className="gg-play-button-r" />
+        <TextWrap1>Play</TextWrap1>
+      </NavButton>
+      <NavButton as={NavLink} to="/movies">
+        <MoviesIcon className="gg-film" />
+        <TextWrap2>Movies</TextWrap2>
+      </NavButton>
+    </Nav>
+  )
+}
+
+const Nav = styled.nav`
+  position: fixed;
+  overflow-y: scroll;
+  bottom: 0;
+  display: flex;
+  gap: 10px;
```

Comment:
Be careful with `gap` for flexbox as it's not fully supported by all browsers yet!